src/BrowscapSiteConsole.php

```diff
@@ -12,13 +14,18 @@ public function __construct()
     {
         parent::__construct('Browscap Website', 'dev-master');
 
-        // @todo Config should come from a shared container, this is terrible
-        $config = require __DIR__ . '/../config/config.php';
+        /** @var ContainerInterface $container */
+        $container = require __DIR__ . '/../config/container.php';
+
+        $dbConfig = $container->get('Config')['db'];
 
         $commands = [
             new Command\GenerateStatisticsCommand(new AnalyseStatistics(
-                new \PDO($config['db']['dsn'], $config['db']['user'], $config['db']['pass'])
+                new \PDO($dbConfig['dsn'], $dbConfig['user'], $dbConfig['pass'])
             )),
+            new Command\GenerateBuild(
+                (new BuildGeneratorFactory())->__invoke($container)
```

Comment:
Is it possible to write in the container config: `BuildGenerator::class => BuildGeneratorFactory::class` so that we may write here: `$container->get(BuildGenerator::class);`